src/main/java/com/programmers/ticketparis/controller/schedule/ScheduleController.java

```diff
@@ -0,0 +1,42 @@
+package com.programmers.ticketparis.controller.schedule;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.programmers.ticketparis.dto.ApiResponse;
+import com.programmers.ticketparis.dto.schedule.request.ScheduleCreateRequest;
+import com.programmers.ticketparis.dto.schedule.response.ScheduleResponse;
+import com.programmers.ticketparis.service.schedule.ScheduleService;
+
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/performances/{performanceId}")
+public class ScheduleController {
+
+    private final ScheduleService scheduleService;
+
+    @PostMapping("/schedules")
+    @ResponseStatus(HttpStatus.CREATED)
+    public ApiResponse<ScheduleResponse> createSchedule(@PathVariable Long performanceId, @Valid @RequestBody ScheduleCreateRequest request, HttpServletRequest httpServletRequest) {
```

Comment:
`p4`
파라미터가 너무 길어지면 한눈에 보기 힘들 수 있으니 적절히 띄워주면 좋을것 같습니다.
```suggestion
    public ApiResponse<ScheduleResponse> createSchedule(
        @PathVariable Long performanceId, 
        @Valid @RequestBody ScheduleCreateRequest request, HttpServletRequest httpServletRequest
    ) {
```

추가로 hard wrap에 대한 code style 컨벤션도 정했나요?

Reply:
반영: [313ac9a](https://github.com/prgrms-be-devcourse/BE-04-TICKETPARIS/pull/30/commits/313ac9afb4503e98fe2d153024cda2aff1dafe32)
- hard wrap을 120으로 가기로 다같이 정했습니다. 관련해서 수정도 했습니다!